src/main/python/gui/gui.py

```diff
@@ -41,6 +41,9 @@ class StartingMenu(tk.Frame):
     def __init__(self, master):
         tk.Frame.__init__(self, master)
         global user
+        global proj_manager
```

Comment:
Why use a global instead of making this a field on the guy class?

Reply:
Right now, gui isnt a class, but an actual application. I'll definitely look into that though.